src/main/java/spoilagesystem/timestamp/LocalTimeStampService.java

```diff
@@ -133,15 +133,19 @@ public OffsetDateTime getTimeStamp(ItemStack item) {
 
     @Nullable
     private OffsetDateTime parseExpiryFromPersistentData(ItemMeta meta) {
-        String expiryString = meta.getPersistentDataContainer().get(expiryKey, STRING);
-        if (expiryString != null) {
-            try {
-                return OffsetDateTime.parse(expiryString, ISO_OFFSET_DATE);
-            } catch (DateTimeParseException exception) {
-                plugin.getLogger().log(SEVERE, "Failed to parse expiry from persistent data container", exception);
+        try {
+            String expiryString = meta.getPersistentDataContainer().get(expiryKey, STRING);
+            if (expiryString != null) {
+                try {
+                    return OffsetDateTime.parse(expiryString, ISO_OFFSET_DATE);
+                } catch (DateTimeParseException exception) {
+                    plugin.getLogger().log(SEVERE, "Failed to parse expiry from persistent data container", exception);
```

Comment:
This line is still going to log exceptions - you want to remove the outer try/catch and ignore the exception rather than logging it here. The exception isn't bubbling up so the outer block won't catch anything.

Reply:
That makes sense, thanks for explaining that!